README.md

```diff
@@ -43,6 +43,7 @@ Both scripts must be run from the Simplified-iOS repo root.
 
 # Building Without Adobe DRM
 
+**Note:** In order to build dependencies, you need Xcode 11.4 or higher.
```

Comment:
can you mention the Xcode 11.4 requirement above where System Requirements is? This applies to both DRM and non-DRM builds.